WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -76,6 +76,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
+      <h2><%= anchorable "Regulations and Guidelines" %></h2>
+      <% @regulations.each do |reg| %>
+        <div>
+          <%= link_to reg["url"] do %>
+            <h4><%= wca_highlight(reg["id"], @omni_query) %></h4>
```

Comment:
Here we can just make use of [this variable](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/views/search_results/index.html.erb#L9). `wca_highlight(reg["id"], query_parts)`